### ISSUE TITLE
21.0.0 Crash when left on Movies or TV Shows - Linux 6.1.92-1-MANJARO

### ISSUE THREAD

**Issue by Phaelkyn** [xbmc/xbmc#25341]
### Describe the bug
Here is a clear and concise description of what the problem is:

Kodi will crash if left on Movies or TV Shows. This started after upgrading to 21.
If I move quickly after start up to another main menu then it will work. As soon as I highlight Movies/TV Shows it will crash in 2 or 3 seconds. 

This has happened on other machines, too, after the 21 upgrade. Android FireTV specifically. 

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

Normal operation.

## Actual Behavior

Highlighting Movies or TV Shows will cause Kodi to become non responsive then it will close. This happens in just a couple of seconds. 

### To Reproduce

1. Run Kodi from GUI or Terminal
2. Keep highlighted on Movies or move to TV Shows
3. Kodi will be non responsive in a second or two then immediately close

### Debuglog

The debuglog can be found here:
[kodi_crashlog-20240617_111714.log](https://github.com/user-attachments/files/15873392/kodi_crashlog-20240617_111714.log)
[kodi_crashlog-20240617_112445.log](https://github.com/user-attachments/files/15873458/kodi_crashlog-20240617_112445.log)

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

When running from the terminal, it lists some permission denied messages.
` kodi --debug                                ✔  51m 22s  
/usr/bin/kodi: line 215: 52005 Aborted                 (core dumped) ${KODI_BINARY} ${ENV_ARGS} $SAVED_ARGS
find: ‘/home/dj/.cache/yay/libavresample/pkg’: Permission denied
find: ‘/home/dj/.cache/yay/libdvbcsa-git/pkg’: Permission denied
find: ‘/home/dj/.cache/yay/spotube/pkg’: Permission denied
find: ‘/home/dj/ardour-git/pkg’: Permission denied
`

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:  Arch: x86_64
 Kernel: Linux 6.1.92-1-MANJARO #1 SMP PREEMPT_DYNAMIC Mon May 27 03:37:00 UTC 2024
 Release: Manjaro Linux
 - Kodi version: package manager shows 21.0-4 but inside Kodi/System Information it shows 21.0.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by neo1973** [comment]
I don't think we can do anything here at the moment. The stack trace and the log (`ffmpeg[0x5576a28dfd70]: Assertion frame->buf[0] failed at src/libavcodec/decode.c:471`) suggest that this is a problem internal to FFmpeg. Maybe when we switch to a newer FFmpeg (#24972) this will be fixed.

**Comment by Phaelkyn** [comment]
@neo1973 would you speculate that this is a distro specific issue? 

One thought that I had is that maybe kodi needs to be added to the audio group based on the messages in the terminal.
```
kodi --debug                                IOT ✘  1m 8s    
/usr/bin/kodi: line 215: 455845 Aborted                 (core dumped) ${KODI_BINARY} ${ENV_ARGS} $SAVED_ARGS
find: ‘/home/dj/.cache/yay/libavresample/pkg’: Permission denied
find: ‘/home/dj/.cache/yay/libdvbcsa-git/pkg’: Permission denied
find: ‘/home/dj/.cache/yay/spotube/pkg’: Permission denied
find: ‘/home/dj/ardour-git/pkg’: Permission denied
Crash report available at /home/dj/kodi_crashlog-20240730_103933.log
```

These messages generated when I tried to run a media update. All of these are audio applications that are completely unrelated to Kodi but run in Pipewire.

**Comment by Phaelkyn** [comment]
UPDATE. I purged this source from the library and it no longer crashes. It no longer has anything in the library either, however. 
When I added the source back, which is an internal SSD, the same crash occurred again. Current workaround is to remove all videos from Movies and TV Shows. 
Music library is working just fine from the same SSD.
